zeekctl/zeekctl.spec

```diff
@@ -0,0 +1,159 @@
+Name:           zeekctl
+Version:        2.0.0
+Release:        1%{?dist}
+Epoch:          1
+Summary:        Tool for managing Zeek deployments.
+
+License:        BSD
+URL:            https://github.com/zeek/zeekctl
+Source0:        https://www.zeek.org/downloads/%{name}-%{version}.tar.gz
+Source1:        zeek.service
+
+Provides:       broctl
+
+BuildRequires:  cmake >= 2.6.3
+BuildRequires:  gcc-c++
+BuildRequires:  python2-devel
+BuildRequires:  capstats >= 0.26
+BuildRequires:  trace-summary >= 0.88
+BuildRequires:  systemd
+BuildRequires:  swig
+BuildRequires:  libpcap-devel
+BuildRequires:  bro-core >= 2.6.1
```

Comment:
This should be 2.6.3. It's not a circular dependency. Bro should be built first.

Reply:
Fixed